examples/sim_fibo_ad_cp.cpp

```diff
@@ -265,8 +265,8 @@ try
         fis_solver.reset(new NewtonIterationBlackoilSimple(param, parallel_information));
     }
 
-    // Write parameters used for later reference.
-    bool output = param.getDefault("output", true);
+    // Write parameters used for later reference. (only if rank is zero)
+    bool output = param.getDefault("output", true) && ( grid->comm().rank()==0 );
```

Comment:
Switching the order (doing the rank check first) would eliminate another minor source of terminal output mess.